lib/libc/pthread/pthread_mutexattr_setprotocol.c

```diff
@@ -80,25 +80,23 @@
  *
  ****************************************************************************/
 
-int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr,
-				  int protocol)
+int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr, int protocol)
```

Comment:
This file has changes for formatting and these changes are not matched with commit message. Let's split commits.